packages/create-remix/templates/arc-stack/README.md

```diff
@@ -0,0 +1,72 @@
+   - Next, run `aws configure` and paste in your credentials.
+
+## Development
+
+```sh
+echo "NODE_ENV=development" >> .env
+echo SESSION_SECRET=$(openssl rand -hex 32) >> .env
+npm run dev
+```
+
+> If you don't have openssl installed, you can also use [1password][generate_password] to generate a random secret, just replace `$(openssl rand -hex 32)` with the generated secret.
+
+This starts your app in development mode, rebuilding assets on file changes.
+
+This is a pretty simple note-taking app, but it's a good example of how you can build a full stack app with Architect and Remix. The main functionality is creating users, logging in and out, and creating and deleting notes.
+
+### Relevant code:
+
+- creating users, and logging in and out [./app/models/user.server.ts](./app/models/user.server.ts)
+- user sessions, and verifying them [./app/session.server.ts](./app/session.server.ts)
+- creating, and deleting notes [./app/models/note.server.ts](./app/models/note.server.ts)
+
+The database that comes with `arc sandbox` is an in memory database, so if you restart the server, you'll lose your data. Production environments won't behave this way.
+
+## Deployment
+
+This Remix Stack comes with two GitHub actions that handle automatically deploying your app to production and staging environments.
+
+Prior to your first deployment, you'll need to do a few things:
+
+- Create a new [GitHub repo](https://repo.new)
+
+- Make sure you have your `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` saved to your GitHub repo's secrets. You can re-use the ones you used for the AWS CLI or you can go to your AWS [security credentials][aws_access_key_id] and click on the "Access keys" tab, and then click "Create New Access Key", then you can copy those and add them to your repo's secrets.
+
+Along with your AWS credentials, you'll also need to give your CloudFormation a `SESSION_SECRET` variable of its own for both staging and production environments.
+
+```sh
+arc env staging SESSION_SECRET $(openssl rand -hex 32)
+arc env production SESSION_SECRET $(openssl rand -hex 32)
+```
+
+> If you don't have openssl installed, you can also use [1password][generate_password] to generate a random secret, just replace `$(openssl rand -hex 32)` with the generated secret.
+
+## Where do I find my CloudFormation?
+
+You can find the CloudFormation template that Architect generated for you in the sam.yaml file.
+
+To find it on AWS, you can search for [CloudFormation][cloudformation] (make sure you're looking at the correct region!) and find the name of your stack that matches what's in `app.arc`
+
+To find your deployed api, you can search for [APIGateway][apigateway] (make sure you're looking at the correct region!)
+
+To find your deployed lambda functions, you can search for [Lambda][lambda] (make sure you're looking at the correct region!)
```

Comment:
I might mention that the created CloudFormation "Stack" will be named `RemixAwsStackStaging` based on the `app.arc`'s `@app` entry. Also a user can access all resources from that stack in the CloudFormation "Resources" tab.
I use this to quickly navigate to Lambdas and then to logs since it automatically goes to the correct region.